docs/faq.md

```diff
@@ -0,0 +1,7 @@
+# FAQ
+
+> Argo CloudOps server returns "INTERNAL ERROR" when following logs
+
+This happens when a deployment executes for a long time. Just reissue the logs command to retry.
```

Comment:
Should we mention that we're currently looking into the issue?

Reply:
maybe file a public issue and link?

Reply:
Added a public issue & linked in the faq:

https://github.com/argoproj-labs/argo-cloudops/issues/93